Gifski/util.swift

```diff
@@ -1030,8 +1030,11 @@ extension CGSize {
 
 	func aspectFit(to boundingSize: CGSize) -> CGSize {
 		let ratio = min(boundingSize.width / width, boundingSize.height / height)
+		return self * ratio
+	}
 
-		var size = self * ratio
+	func aspectFit(to scale: CGFloat) -> CGSize {
```

Comment:
I don't understand what you're trying to achieve here?

`scale` is not the correct word to use here. It's not a scale you're passing it. It could be called `sides` or `widthHeight`.

Reply:
If you want to add a new utility method, please add a good description of its purpose to the code too.